src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/AWSSecretsManagerPlugin.java

```diff
@@ -240,6 +242,12 @@ private boolean updateSecret(boolean forceReFetch) throws SQLException {
     return fetched;
   }
 
+  private void attemptToUpdateCurrentHostInfoWithSecrets(HostInfo hostInfo) {
+    HostInfo currentHostInfo = this.currentConnectionProvider.getCurrentHostInfo();
+    this.currentConnectionProvider.setCurrentConnection(this.currentConnectionProvider.getCurrentConnection(),
+        ConnectionUtils.copyWithAdditionalProps(currentHostInfo, hostInfo));
```

Comment:
super nit:
If we are breaking the parameters due to line limit, I prefer we put each parameter on a new line, i.e.
```java
    this.currentConnectionProvider.setCurrentConnection(
        this.currentConnectionProvider.getCurrentConnection(),
        ConnectionUtils.copyWithAdditionalProps(currentHostInfo, hostInfo));
```

Reply:
Done!